### PR TITLE
Feature/Component-Overview: Update SignUp documentation

### PR DESCRIPTION
**Pull request by Danielius1922**

### CODE REVIEW

File: content/en/architecture/component-overview.md [hunk 1]
```diff
@@ -68,24 +71,20 @@ hide footbox
 participant D as "Device"
```
Comment (ondrejtomcik) [inline content/en/architecture/component-overview.md:71]: please include participant O as "OAuth 2.0 Server"

File: content/en/architecture/component-overview.md [hunk 2]
```diff
@@ -68,24 +71,20 @@ hide footbox
...
 participant CGW as "CoAP Gateway"
 participant AS as "Authorization Server"
-participant RA as "Resource Aggregate"
 
 D -> CGW ++: Sign Up
 group OAuth2.0 Authorization Code Grant Flow
-    CGW -> AS ++: Exchange authorization code for access token
-    AS -> AS: Verify Authorization Code
-    return Ok\n(Access Token, Refresh Token, ...)
+    CGW -> CGW: Verify and exchange authorization code for access token
```
Comment (ondrejtomcik) [inline content/en/architecture/component-overview.md:77]: Exchange is between CGW and O

File: content/en/architecture/component-overview.md [hunk 3]
```diff
@@ -68,24 +71,20 @@ hide footbox
...
 end
-CGW -> RA ++: Register device resource
-return Registered
+CGW -> AS ++: Add device resource
```
Comment (ondrejtomcik) [inline content/en/architecture/component-overview.md:79]: CGW Registers device, or Register user's device

Comment (ondrejtomcik) [reply content/en/architecture/component-overview.md]: This wasn’t resolve. Coap gw does not registers resource. It’s only device, creating record that device is connected and assigning it to a iser.

**Review by ondrejtomcik — APPROVED** [review]

File: content/en/architecture/component-overview.md (hunk 3 above)
Comment (Danielius1922) [reply content/en/architecture/component-overview.md]: Sorry, I've read this in the morning and forgot about this. Thanks for updating it.

**Pull request merged**